packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -537,6 +539,11 @@ export interface RawResolversConfig extends RawConfig {
    * @description Turning this flag to `true` will generate resolver signature that has only `resolveType` for interfaces, forcing developers to write inherited type resolvers in the type itself.
    */
   onlyResolveTypeForInterfaces?: boolean;
+  /**
+   * @description Make __typename of resolver mappings non-optional
+   * @default false
+   */
+  resolversNonOptionalTypename?: boolean | ResolversNonOptionalTypenameConfig;
```

Comment:
Iinstead of adding it to the shared `nonOptionalTypename`, I chose to add this new option because it's only relevant to the `__typename` in types generated by `typescript-resolvers`